docs/07-how_to_run_a_node/03-run_using_docker/04-run-validator-node.md

```diff
@@ -0,0 +1,212 @@
+
+At the end of the session the script would populate directory `deployments/validator-node/`*`network`* with the proper files and you would get a message similar to:
+
+```bash
+=== Run the compose project with the following command: 
+
+========================
+docker compose -f /home/your_user/compose-zkverify-simplified/deployments/validator-node/testnet/docker-compose.yml up -d
+========================
+```
+
+Before actually launching the node, you can further inspect and customize the execution by manually editing `deployments/validator-node/`*`network`*`/.env` file. Entries under `# Node miscellaneous` section are related to the Docker container, while those under `# Node config` section are related to the Substrate node instance. Pay attention to only take this action if you fully understand its implications.
+
+## Run the Node
+
+*Time to start now!* Within the terminal type the command proposed by the script:
+
+```bash
+docker compose -f /home/your_user/compose-zkverify-simplified/deployments/validator-node/testnet/docker-compose.yml up -d
+```
+
+*and you are done!* This allows you to start the node in background, for checking it is running properly you can type:
+
+```bash
+docker container ls
+```
+
+and you should get something similar to:
+
+```bash
+CONTAINER ID   IMAGE                            COMMAND                CREATED              STATUS              NAMES
+ca4bdf2c6f05   horizenlabs/zkverify:0.2.0-rc1   "/app/entrypoint.sh"   About a minute ago   Up About a minute   validator-node
+```
+
+showing your node has started correctly.
+
+Up to here your validator node is running properly but, in order to make it eligible for authoring new blocks, you need to follow through with a few additional steps.
+
+## Next Steps
+
+In this section you can learn how to register a new validator on the blockchain. The operations described below must be performed just once and they consist in the submission of some extrinsics (transactions, in Substrate terminology) resulting eventually in your node being able to author new blocks and consequently earn new tokens through staking mechanism. Note that since you are going to submit extrinsics which changes the blockchain state, you need sufficient funds in the account (uniquely identified by your secret phrase) associated with your validator so that you can pay transaction fees; if that's not your case, it is mandatory you send some tokens to it.
+
+For security reasons your validator node does not expose an RPC interface but you need a user friendly way for submitting the extrinsics, so the first thing to do is to temporarily run an additional RPC node (refer to [this page](./02-run-rpc-node.md)) which you can shutdown once you have completed everything.
+
+Make sure both your validator node as well as your RPC node are running using command:
```

Comment:
Unless unsafe RPC methods are needed it would be a lot easier to just connect using the public RPC.

https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ftestnet-rpc.zkverify.io#/explorer

Reply:
I agree with you, we are not using unsafe methods so it is fine to go for public RPC.
I've modified accordingly.